CMakeLists.txt

```diff
@@ -311,13 +356,13 @@ if(NETWORKIT_PYTHON)
 			endif()
 		endif()
 
-	target_link_libraries(${TARGET_LIB} PRIVATE OpenMP::OpenMP_CXX PUBLIC tlx)
-		if(${TARGET_LIB} STREQUAL stat)
-			install(TARGETS ${TARGET_LIB}
-				LIBRARY DESTINATION "${NETWORKIT_LIB_DEST}/networkit/profiling")
-		else()
-			install(TARGETS ${TARGET_LIB}
-				LIBRARY DESTINATION "${NETWORKIT_LIB_DEST}/networkit")
+		target_link_libraries(${TARGET_LIB} PRIVATE OpenMP::OpenMP_CXX PUBLIC tlx)
+		install(TARGETS ${TARGET_LIB}
+			LIBRARY DESTINATION "${NETWORKIT_LIB_DEST}/networkit")
+
+		if(WIN32)
+			FIND_PACKAGE(Python3 COMPONENTS Interpreter Development)
+			target_link_libraries(${TARGET_LIB} PRIVATE ${Python3_LIBRARIES})
 		endif()
```

Comment:
Why is this in `if(WIN32)`? Don't we need to link against Python libs on Linux/macOS?

Reply:
Normally they are found by default. But maybe it is a good idea to use it on all systems.